Cargo.toml

```diff
@@ -8,6 +8,5 @@ crate-type = ["cdylib"]
 name = "lambda"
 
 [dependencies]
-cpython = "0.1"
-crowbar = "0.2"
-python3-sys = { version = "0.1.3", features = ["python-3-4"], optional = true }
\ No newline at end of file
+crowbar = { git = "https://github.com/softprops/rust-crowbar.git", branch = "re-export-cpython-macros" }
+python3-sys = { version = "0.2.1", features = ["python-3-4"], optional = true }
```

Comment:
the rustc flag for this gets passed in [here](https://github.com/softprops/serverless-crowbar/blob/efc7626c0be225eeb0dce4abc3f14461adbe6c14/serverless.yml#L9-L10)